Dockerfile

```diff
@@ -1,40 +1,39 @@
-FROM amazonlinux:2.0.20200722.0
+FROM ubuntu:focal-20210401
 
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
Because we're installing postgresql we need to tell it that we don't care what the setup values are. We can remove this when we no longer install all of postgres.